dash-pipeline/tests/saithrift/ptf/acl/test_saithrift_acl.py

```diff
@@ -0,0 +1,231 @@
+        self.meta = copy.copy(self.__dict__)
+        del self.meta["saithrift"]
+
+    def runTest(self):
+        inner_pkt = simple_udp_packet(eth_src=self.saithrift.eni_mac,
+                                      ip_dst=self.saithrift.dst_ca_ip,
+                                      ip_src=self.sip)
+        vxlan_pkt = simple_vxlan_packet(eth_dst=self.saithrift.our_mac,
+                                        ip_dst=self.saithrift.vip,
+                                        ip_src=self.saithrift.src_vm_pa_ip,
+                                        udp_sport=11638,
+                                        with_udp_chksum=False,
+                                        vxlan_vni=self.saithrift.outbound_vni,
+                                        inner_frame=inner_pkt)
+        inner_exp_pkt = simple_udp_packet(eth_dst=self.saithrift.dst_ca_mac,
+                                          eth_src=self.saithrift.eni_mac,
+                                          ip_dst=self.saithrift.dst_ca_ip,
+                                          ip_src=self.sip)
+        vxlan_exp_pkt = simple_vxlan_packet(eth_dst="00:00:00:00:00:00",
+                                            eth_src="00:00:00:00:00:00",
+                                            ip_dst=self.saithrift.dst_pa_ip,
+                                            ip_src=self.saithrift.vip,
+                                            udp_sport=0,
+                                            with_udp_chksum=False,
+                                            vxlan_vni=self.saithrift.vnet_vni,
+                                            inner_frame=inner_exp_pkt)
+        vxlan_exp_pkt[IP].chksum = 0
+        vxlan_exp_pkt[IP][UDP][VXLAN].flags = 0
+
+        pkt_exp = vxlan_exp_pkt
+        print("Sending packet...\n", vxlan_pkt.__repr__())
+        send_packet(self.saithrift, 0, vxlan_pkt)
+        print("\n")
+        if self.exp_receive:
+            print("Verifying packet...\n", pkt_exp.__repr__())
+            verify_packet(self.saithrift, pkt_exp, 0)
+        else:
+            print("Verifying drop...")
+            verify_no_other_packets(self.saithrift)
+        print("\n")
+        print("Acl test {} OK".format(self.meta))
+
+
+class TestSaiThrift_dash_acl(ThriftInterfaceDataPlane):
+    """ Test saithrift DASH ACL"""
+
+    def create_entry(self, create_func, remove_func, entry, *args, **kwargs):
+        status = create_func(self.client, entry, *args, **kwargs)
+        assert (status == SAI_STATUS_SUCCESS)
+        self.teardown_stack.append((remove_func, entry))
+        return status
+
+    def create_obj(self, create_func, remove_func, *args, **kwargs):
+        obj = create_func(self.client, *args, **kwargs)
+        assert (obj != SAI_NULL_OBJECT_ID)
+        self.teardown_stack.append((remove_func, obj))
+        return obj
+
+    def setUpSwitch(self):
+        self.switch_id = 5
```

Comment:
This should be already defined once you have switched to `SaiHelperSimplified` as Chris wrote.